app/models/setting/netbox.rb

```diff
@@ -20,7 +20,10 @@ def self.load_defaults
       return unless super
 
       transaction do
-        default_settings.each { |s| create! s.update(category: 'Setting::Netbox') }
+        default_settings
+          .map { |s| s.merge(category: 'Setting::Netbox') }
+          .map { |s| s.slice(*column_names.map(&:to_sym)) }
+          .map { |s| find_or_initialize_by(s.slice(:name)).update!(s) }
```

Comment:
Should this be `.each`?

Reply:
it doesn't really matter in this case, both iterate over the array

Reply:
yeah - I just find it weird when `map` changes external data.

Reply:
it doesn't really change the external data, but creates a new array containing the values returned by the block